css/properties/columns.json

```diff
@@ -0,0 +1,150 @@
+                "version_added": true
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "52"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "9"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "52"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "22"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": "10"
+            },
+            "opera": [
+              {
+                "version_added": "11.10"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "15"
+              }
+            ],
+            "opera_android": [
+              {
+                "version_added": "11.5"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "32"
+              }
+            ],
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "3"
+            },
+            "safari_ios": {
+              "prefix": "-webkit-",
+              "version_added": "3.2"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "on_display_table_caption": {
+          "__compat": {
+            "description": "On <code>display: table-caption</code>",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "37"
+              },
+              "firefox_android": {
+                "version_added": "37"
+              },
+              "ie": {
+                "version_added": true
+              },
+              "ie_mobile": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            }
```

Comment:
Status is missing. Is this a standard behaviour?

Reply:
Another "on `display: table-caption`" case. Again, I don't know how to handle the status statement on this subfeature either.

Reply:
It is standard, and non-experimental.

Reply:
OK, added.